astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -24,6 +25,18 @@
 ) as dag:
     start = DummyOperator(task_id="start")
 
+    # Execute AWS command then sleep for 5 min so that cluster would be available
+    create_redshift_cluster = BashOperator(
+        task_id="create_redshift_cluster",
+        bash_command="aws redshift create-cluster "
+        "--db-name dev "
+        "--cluster-identifier redshift-cluster-1 "
```

Comment:
lets make the cluster identifier something like astro-providers-cluster

Reply:
Changed the cluster name from `redshift-cluster-1` to `astro-providers-cluster` also parameterize it